workshop/content/docs/introduction.md

```diff
@@ -44,9 +45,16 @@ The above examples may help as starting points for your own pygeoapi setup and d
 ## History
 
 Starting in 2018, pygeoapi emerged as part of the initial efforts for the development of OGC API standards. OGC API
-code sprints were instrumental for agile development and pouring the foundation of the project. The core design principles
-of the project were and are modularity, extensibility, building by exception, building on a large ecosystem of Free Open Source and OSGeo
-components such as GDAL, rasterio, Shapely, Pandas, Elasticsearch, PostGIS and many others.
+code sprints were instrumental for agile development and pouring the foundation of the project.
+
+The core design principles are as follows:
+
+- simplicity / low barrier to entry
+- long term sustainability
+- modularity
+- extensibility
+- building by exception based on a large ecosystem of Free Open Source and OSGeo
```

Comment:
*"..building by exception based on a large ecosystem of Free Open Source and OSGeo..."*
Maybe I don't understand what "building by exception" means, but would simply phrase: 
*"..building on a large ecosystem of Free and Open Source and OSGeo..."*